backend/index.js

```diff
@@ -43,6 +44,7 @@ app.use('/', googleMapsRoute);
 app.use('/', registrationRoute);
 app.use('/', educationRoute);
 app.use('/', messageRoute);
+app.use('/', domainOfActivityRoute);
```

Comment:
I would not create another route for this.. Just as I merged the "majors", "institutions", and "degrees" in the same route component (education route), we should also add the domain of activity endpoint in the route that is closest to it as a concept, which is "employerRoute". This domain of activity is strictly related to the employer and we don't use it in any other entity, so it's okay if we add it there.

Reply:
To be honest I don't really see the point in changing this: I see why you would do it in a different way, but I preferred to have a separate route for more clarity, and if there is no real problem with that and it's just personal taste I think I should focus on more functional changes for now since there are a lot of comments here and some more on plad-78, and we have to close the sprint at some point. Also, to be honest I think it made more even sense for the majors and institutions to have a separate route, since they are used in the placements without the education entity, but still this is personal taste I think.

Reply:
I understand your point, however I think that we should keep the routes grouped as much as possible in how they make sense. Another example: we have a file for "locationRoutes" and another for "googleMapsRoutes". That's just wrong, since it's the same concept, that of the location. It's independent of the implementation (googleMaps API), and it should not be split in 2 separate files.

And the educationRoute is not for accessing only the education table, it's a general concept that englobes a series of routes that have meaning only in that context. 

As for the "domainOfActivityRoute", if we leave this here, it's not going to break the code, but I don't see how it makes our structure better, since it's a file that contains only 1 route that only makes sense in the employer's context.